cinemaAppFE/src/app/home/index.ts

```diff
@@ -0,0 +1 @@
+export * from './home.component';
```

Comment:
usualy index.ts files are used when there are multiple components in the same folders, ot should work without it in this case. This is a small observation.